Dockerfile

```diff
@@ -10,25 +10,26 @@ LABEL org.label-schema.schema-version="1.0" \
     org.label-schema.vendor="Alfresco"
 
 RUN yum -y update \
-    yum-utils-1.1.31-52.el7 \
-    yum-plugin-ovl-1.1.31-52.el7 \
-    yum-plugin-fastestmirror-1.1.31-52.el7 \
-    bind-license-9.11.4-9.P2.el7 \
-    glibc-2.17-292.el7 \
+    yum-utils-1.1.31-53.el7 \
+    yum-plugin-ovl-1.1.31-53.el7 \
+    yum-plugin-fastestmirror-1.1.31-53.el7 \
+    bind-license-9.11.4-16.P2.el7_8.2 \
+    glibc-2.17-307.el7.1 \
+    systemd-219-73.el7_8.5 \
+    expat-2.1.0-11.el7 \
+    bash-4.2.46-34.el7 \
+    shared-mime-info-1.8-5.el7 \
+    libxml2-2.9.1-6.el7.4 \
     openssl-libs-1.0.2k-19.el7 \
-    krb5-libs-1.15.1-37.el7_7.2 \
-    setup-2.8.71-10.el7 \
-    python-2.7.5-86.el7 \
+    krb5-libs-1.15.1-46.el7 \
+    setup-2.8.71-11.el7 \
+    python-2.7.5-88.el7 \
     gnupg2-2.0.22-5.el7_5 \
     nss-3.44.0-7.el7_7 \
-    nss-sysinit-3.44.0-7.el7_7 \
```

Comment:
not needed since brought by main nss, libssh brought by curl, systemd-libs by systemd